### DISCUSSION TITLE
Doris 数组Array 存储与查询性能问题

### DISCUSSION THREAD

**Discussion by LCehoennardo** [apache/doris#16541] [A - General / Q&A]
请问一下Doris数组类型的查询性能如何？

比如，在一个 3 FE，3BE的集群上，构建一个数据量十几亿、单个数组最大长度可能达到 1 千多的表，进行`ARRAY_INTERSECT`（集合交集）计算，过滤出所有交集不为空的数据，大概需要多长时间，是毫秒级、秒级、还是需要十几秒呢？

或者，有没有一些Doris数组类型的存储、性能方面的文章，我找了很久没找到，只看到关于如何使用的介绍。

谢谢！

**Comment by LemonLiTree** [comment]
目前应该是没测过，不过如果大佬感兴趣的话我们可以细聊下

**Reply by LCehoennardo** [reply]
谢谢大佬。不过现在不用了，我们用bitmap解决了，以后有需要在咨询您

**Reply by LemonLiTree** [reply]
bitmap is very GOOD
